app/views/contexts/thank_you_context.py

```diff
@@ -15,6 +17,7 @@ def build_thank_you_context(
     session_data: SessionData,
     submitted_at: datetime,
     survey_type: str,
+    schema: QuestionnaireSchema,
```

Comment:
`schema` should be passed as one of the first arguments to `build_thank_you_context` to be consistent with how it is passed around elsewhere.